### DISCUSSION TITLE
removing annotations below draw.gam()

### DISCUSSION THREAD

**Discussion by jybarnagaud** [gavinsimpson/gratia#306] [Q&A] [answered]
Dear all

Is there an easy way to remove the lower-right margin annotations showing the basis or effect type below draw.gam outputs, without having to recode all the plot by hand ?

Sorry if this has been already answered or if the solution is obvious, I'm lacking ggplot and github proficiency to find my way myself. 

Thanks!

**Comment by gavinsimpson** [comment]
Not currently, no, but it should be a simple fix to allow `caption` to be passed on from `draw.gam` to the code underlying that does the plotting. The one downside of this is that it will still leave space for the caption, so you'll have a little more whitespace than you might like.

The full fix would be to allow `FALSE` as an option for `caption` in the underlying plotting code. But to allow that I'd need to handle it in the functions for each smooth type and that's a slightly bigger fix right now.

**Reply by gavinsimpson** [reply]
If you install from GitHub now, you can do `draw(m, caption = "")` and that will kill the captions but you'll still get space left for them. I have opened #307 to track the longer term fix of allowing `FALSE` so that no space is left where the caption would have been.

**Reply by jybarnagaud** [reply]
Thanks. the caption = "" had no effect on my code, nor on the sample code of the draw.gam function. No big deal, can wait until a fix later.

**Reply by gavinsimpson** [reply]
@jybarnagaud Did you install the from GitHub? `caption = ""` won't work with the released version as it needed `...` to be passed on from `draw.gam` to the underlying plotting code and in the released version it wasn't doing that.

**Reply by jybarnagaud** [reply]
@gavinsimpson yes I reinstalled gratia from your github before trying - the install looked successful but still caption = "" didn't work.

**Reply by gavinsimpson** [reply]
That's odd; it worked for me. Let me just confirm I actually pushed that change to GitHub...

**Reply by gavinsimpson** [reply]
Sorry @jybarnagaud; I am editing code in three different packages right now and I had actually forgotten to push the fix I mentioned. It is pushed now.

**Reply by jybarnagaud** [reply]
looks good now, thanks!

**Reply by ewilsonloise** [reply]
great fix, thank you @gavinsimpson

**Comment by gavinsimpson** [comment, marked as answer]
You can now suppress captions properly (i.e. the space where the caption would be disappears now) using the `caption` argument to `draw.gam`.
